src/Maestro/maestro-angular/src/maestro-client/models.ts

```diff
@@ -1,6 +1,7 @@
 import { parseISO } from "date-fns";
 
 import { Helper } from "./helper";
+import { THIS_EXPR } from '@angular/compiler/src/output/output_ast';
```

Comment:
We (thankfully) don't seem to be using `THIS_EXPR` in thsi code. Can we remove this scary sounding import?

Reply:
I'm not even sure how that got added. Removed.